src/pages/CropSidebar/CropSidebar.js

```diff
@@ -81,9 +81,7 @@ const CropSidebar = ({
     });
     return sidebarStarter;
   });
-  const section = window.location.href.includes('species-selector') ? 'selector' : 'explorer';
-  const sfilters = filterStateRedux[section];
-  // const dictionary = [];
+  const { filters } = filterStateRedux;
```

Comment:
can you not destructure filterStateRedux here and get the actual value from redux above and then rename filters to whatever you name that variable